iocage/lib/Jail.py

```diff
@@ -1684,35 +1681,35 @@ def _configure_nameserver(self) -> None:
     def _configure_localhost_commands(self) -> typing.List[str]:
         return ["ifconfig lo0 localhost"]
 
-    def _limit_resources(self) -> None:
+    def _get_resource_limits_commands(self) -> typing.List[str]:
+
+        commands: typing.List[str] = []
 
         if self.config['rlimits'] is False:
             self.logger.verbose("Resource limits disabled")
-            return
+            return commands
 
         for key in iocage.lib.Config.Jail.Properties.ResourceLimit.properties:
             try:
                 rlimit_prop = self.config[key]
+                if rlimit_prop.is_unset is True:
+                    continue
             except KeyError:
                 continue
-            command = [
+            commands.append(" ".join([
                 "/usr/bin/rctl",
                 "-a",
                 f"jail:{self.identifier}:{key}:{rlimit_prop.limit_string}"
-            ]
-            iocage.lib.helpers.exec(command, logger=self.logger)
+            ]))
+        return commands
 
-    def _release_resource_limits(self) -> None:
+    def _clear_resource_limits(self) -> typing.List[str]:
 
         if self.config['rlimits'] is False:
-            return
+            return []
 
         self.logger.verbose("Clearing resource limits")
-        iocage.lib.helpers.exec(
-            ["/usr/bin/rctl", "-r", f"jail:{self.identifier}"],
-            logger=self.logger,
-            ignore_error=True
-        )
+        return [f"/usr/bin/rctl -r jail:{self.identifier} 2>/dev/null || true"]
```

Comment:
is the || true strictly necessary?

Reply:
The command may exit with 1 but should not raise an error. The other option is to temporary `set -e`. Any preference?

Reply:
*brrrr*

a comment that it may fail and we don't care at this point.